src/openforms/config/models.py

```diff
@@ -624,6 +624,13 @@ def render_privacy_policy_label(self) -> str:
             backend=openforms_backend,
         )
 
+    def render_statement_of_truth_label(self) -> str:
+        return render_from_string(
+            self.statement_of_truth_label,
+            context={"global_configuration": self},
+            backend=openforms_backend,
+        )
```

Comment:
We currently don't support template expressions in this field, so this is not necessary. You can just use `config.statement_of_truth_label` in the template.

Reply:
And how can we avoid the `<p>` tags in the result?If I just use the attribute it renders these as well.